### PR TITLE
awscli 1.16.19

### PR DESCRIPTION
**Pull request by PaigingAlyssa**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/awscli.rb [hunk 1]
```diff
@@ -4,8 +4,8 @@ class Awscli < Formula
   desc "Official Amazon AWS command-line interface"
   homepage "https://aws.amazon.com/cli/"
   # awscli should only be updated every 10 releases on multiples of 10
-  url "https://github.com/aws/aws-cli/archive/1.16.10.tar.gz"
-  sha256 "3082daafec77885043a9eab25f03a641d6d14cedcb10e2cf49ae86a95a939dca"
+  url "https://github.com/aws/aws-cli/archive/1.16.19.tar.gz"
+  sha256 "58fc316e7c8c7d3db39487ca4862ac339743719028e8b9339dd006b92384bd81"
```
Comment (fxcoudert) [inline Formula/awscli.rb]: `awscli should only be updated every 10 releases on multiples of 10`

**Comment by fxcoudert** [comment]
Thanks, but see comment above the `url` line.

File: Formula/awscli.rb (hunk 1 above)
Comment (PaigingAlyssa) [reply Formula/awscli.rb]: @fxcoudert Okay, but when aws-cli sneaks in additional features on a release that doesn't match that criteria, what is one expected to do? Just manually install that version until they roll out another version?

Comment (fxcoudert) [reply Formula/awscli.rb]: There is a new version every day, we simply can't follow that schedule.

**Pull request closed without merging**